src/components/buttons/InterlayCaliforniaOutlinedButton/index.tsx

```diff
@@ -32,10 +33,10 @@ const InterlayCaliforniaOutlinedButton = React.forwardRef<Ref, Props>(
           disabledOrPending
             ? clsx(DISABLED_BORDER_CLASSES, DISABLED_TEXT_CLASSES)
             : clsx(
-                'text-interlayCalifornia',
+                getColorShade('yellow'),
                 'border',
-                'border-interlayCalifornia',
-                'hover:bg-interlayCalifornia',
+                getColorShade('yellow', 'border'),
+                `hover:${getColorShade('yellow', 'bg')}`,
```

Comment:
According to tailwindcss rules, this kind of string interpolation will not work in production.
Re: https://tailwindcss.com/docs/content-configuration#dynamic-class-names
I will not duplicate this comment for the same cases.